### ISSUE TITLE
Static template doesn't show preview

### ISSUE THREAD

**Issue by techmind-git** [codesandbox/sandpack#870]
# Bug report

## Packages affected

-   [ ] sandpack-client
-   [x] sandpack-react

## Description of the problem
I wanted to use the static template and used the basic example
```
import { Sandpack } from '@codesandbox/sandpack-react';

export default function SandpackStaticComponent() {
  const files = {};
  return <Sandpack files={files} theme='dark' template='static' />;
}
```

The result of component is as expected but the preview doesn't show results. It doesn't seem to be able to retrieve the preview
![image](https://user-images.githubusercontent.com/23141619/229463208-131bee88-2134-41e6-b1ec-ff56f3a1af73.png)

![image](https://user-images.githubusercontent.com/23141619/229468253-70fccbf8-017c-4451-a9c3-09df577ec15e.png)

## What were you doing when the problem occurred?
Requesting the page

### What steps can we take to reproduce the problem?
add static component to the page and request the page

I have created a basic [react/vite starter](https://codesandbox.io/p/github/techmind-git/sandpack-react/main) with only the component with static template added to reproduce. With another template it works fine (ie. react).

### Your Environment

| Software                | Name/Version |
| ----------------------- | ------------ |
| Sandpack-client version |   2.2.4           |
| Sandpack-react version  |    2.2.4          |
| Browser                 |  Edge   111.0.1661.62         |
| Operating System        |   Windows 11           |

**Comment by danilowoz** [comment]
@DeMoorJasper, do you have any idea what's going on here?

**Comment by DeMoorJasper** [comment]
Seems like it's passing an empty file object so perhaps there are no files? Can you create a reproducible sandbox for this?

**Comment by techmind-git** [comment]
I have added a link in the issue description to a starter-app in which I could reproduce the issue locally and when imported in codesandbox. Hope this helps to track the cause of the issue

**Comment by DeMoorJasper** [comment]
Fixed in 2.2.9, thanks again for reporting this issue, we totally overlooked this
